tests/python/gpu/test_operator_gpu.py

```diff
@@ -765,30 +765,25 @@ def test_spatial_transformer_with_type():
     check_consistency(sym, ctx_list, grad_req="add")
 
 
-# Checking max pooling consistency over the data sets of different float types is problematic
-# as one max value in a float32 data set may not be the max value in a float16 data set.
-# This function will not be called.
-@with_seed(1234)
-def test_pooling_with_type():
+@with_seed()
+def test_pooling_with_type2():
```

Comment:
Why rename the method since there is no test_pooling_with_type()?

Reply:
There is another `test_pooling_with_type` if you do a bit of search in the file.

Reply:
Sorry, missed it, because this change shows an update on the previous method. It's also more informative to give a more descriptive method name than numerical suffix.